common/scripts/make_image.sh

```diff
@@ -57,7 +57,11 @@ create_cfgfiles ()
 
     mcopy -i  $fatpart_name -o ${GRUB_FS_CONFIG_FILE} ::/grub.cfg
     mcopy -i  $fatpart_name -o ${EFI_CONFIG_FILE}     ::/EFI/BOOT/startup.nsh
-    mcopy -i  $fatpart_name -o ${BSA_CONFIG_FILE}    ::/EFI/BOOT/bsa/bsa.nsh
+    if [ "$BUILD_PLAT" = "SR" ]; then
+        mcopy -i  $fatpart_name -o ${SBSA_CONFIG_FILE}    ::/EFI/BOOT/bsa/sbsa.nsh
+    else
+        mcopy -i  $fatpart_name -o ${BSA_CONFIG_FILE}    ::/EFI/BOOT/bsa/bsa.nsh
```

Comment:
Shouldn't bsa.nsh be copied for SR?

Reply:
sbsa.nsh will take care of running both BSA and SBSA uefi binaries.